spanner/api/Spanner/Program.cs

```diff
@@ -1749,14 +1749,88 @@ await connection.BeginTransactionAsync())
         // [END spanner_dml_batch_update]
 
         // [START spanner_delete_data]
-        public static async Task DeleteSampleDataAsync(
+        public static async Task DeleteIndividualRowsAsync(
+            string projectId, string instanceId, string databaseId)
+        {
+            const int singerId = 2;
+            string connectionString =
+                $"Data Source=projects/{projectId}/instances/{instanceId}"
+                + $"/databases/{databaseId}";
+            List<Album> albums = new List<Album> {
```

Comment:
nit: Bracket on next line unless collection initialization is all in one line. Check here and everywhere.
```suggestion
            List<Album> albums = new List<Album>
            {
```